Software/src/Settings.cpp

```diff
@@ -100,46 +100,62 @@ static const QString NumberOfLeds = "Adalight/NumberOfLeds";
 static const QString ColorSequence = "Adalight/ColorSequence";
 static const QString Port = "Adalight/SerialPort";
 static const QString BaudRate = "Adalight/BaudRate";
+static const QString LedMilliAmps = "Adalight/LedMilliAmps";
+static const QString PowerSupplyAmps = "Adalight/PowerSupplyAmps";
 }
 namespace Ardulight
 {
 static const QString NumberOfLeds = "Ardulight/NumberOfLeds";
 static const QString ColorSequence = "Ardulight/ColorSequence";
 static const QString Port = "Ardulight/SerialPort";
 static const QString BaudRate = "Ardulight/BaudRate";
+static const QString LedMilliAmps = "Ardulight/LedMilliAmps";
+static const QString PowerSupplyAmps = "Ardulight/PowerSupplyAmps";
 }
 namespace AlienFx
 {
 static const QString NumberOfLeds = "AlienFx/NumberOfLeds";
+static const QString LedMilliAmps = "AlienFx/LedMilliAmps";
+static const QString PowerSupplyAmps = "AlienFx/PowerSupplyAmps";
 }
 namespace Lightpack
 {
 static const QString NumberOfLeds = "Lightpack/NumberOfLeds";
+static const QString LedMilliAmps = "Lightpack/LedMilliAmps";
+static const QString PowerSupplyAmps = "Lightpack/PowerSupplyAmps";
```

Comment:
I'm wondering whether this needs to be stored separately per device. Users normally have only one device, and if the defaults are identical across devices, what's the value of having separate keys for this?

Reply:
Well, it's consistent with `NumberOfLeds` which has the same "issue". But I was thinking about looking into a way to have multiple devices, which would also mean multiple devices of the same type, and that'll ~probably~ make current device settings storage inadequate anyway, so I'll leave that as is for now.

Reply:
Well, `NumberOfLeds` has different defaults and limits across devices, but yeah.